.github/workflows/main.yml

```diff
@@ -20,9 +20,10 @@ jobs:
         set -euo pipefail
         mkdir -p "$(dirname $(brew --repo ${{github.repository}}))"
         cp -a "$GITHUB_WORKSPACE" "$(brew --repo ${{github.repository}})"
-        brew install gnu-tar
-        brew install gzip
-        hash -r gzip
+        brew install docker docker-machine
+        brew services start docker-machine
+        docker-machine create --driver virtualbox --virtualbox-boot2docker-url=https://github.com/boot2docker/boot2docker/releases/download/v19.03.5/boot2docker.iso default
+        eval $(docker-machine env default)
```

Comment:
is there a reason this needs to be docker-machine instead of the Docker cask?

Reply:
There is no way to non-interactively give Docker.app the permissions it needs to start a VM sadly. I tried pretty hard and couldn't make anything work. But since these agents already have virtualbox installed and running, using docker-machine is much cheaper than you might expect.